db/migrate/20201029194434_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :user_name
```

Comment:
if Users table has a `user_name` column, does that mean we need a `password` column for the future admin auth part?

Reply:
For now it is just to display as "hey user these are your thoughts" but definitely need password for authentication later.